mappings/net/minecraft/menu/Menu.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS net/minecraft/class_1703 net/minecraft/menu/Menu
+	COMMENT A menu is an object used for synchronizing GUIs between the server and the client.
```

Comment:
Not really synchronizing GUIs but more exactly, synchronizing screens that have item lists (menus).
I'd say:
`A menu is an item list, synchronized between the server and the client to allow updating the client to modify it.`